evm/p2p/state.py

```diff
@@ -36,6 +36,7 @@ class StateDownloader:
     _report_interval = 10  # Number of seconds between progress reports.
     # TODO: Experiment with different timeout/max_pending values to find the combination that
     # yields the best results.
+    # FIXME: Should use the # of peers times MAX_STATE_FETCH here
```

Comment:
Is this as simple as adding a property method for this which computes this value?  If so... sounds like a good issue to create and label with "Good First Issue"

Reply:
It should be, but I plan to tackle it soon when I refactor the `PeerPool` so that a single instance of it can be shared between the state and chain downloaders